assets/aui.selector.publish.js

```diff
@@ -14,15 +14,61 @@
 			fields.each(buildInterface);
 		};
 
+		var updateFilters = function(id,newfilters) {
+			fields.each(function(){
+				var field = $(this); 
+				if (field.attr('id') == "field-" + id){
+					field.data("filters",newfilters);
```

Comment:
String should always be single quoted, not double quoted. In this code, we have both, which is not good.